lib/rubocop/cop/root_cops/use_change.rb

```diff
@@ -0,0 +1,15 @@
+module RuboCop
+  module Cop
+    module RootCops
+      class UseChange < Cop
+        MSG = "Migration must have #change method.".freeze
```

Comment:
Can we add a more descriptive message about why we're enforcing this change?

Reply:
Open to suggestions 😄

Reply:
I'm not sure why we're enforcing this change so I'm not sure what should go here 😬 Looked in the ticket and still don't fully understand

Reply:
Fixed in https://github.com/Root-App/root-ruby-style/pull/31/commits/e5d0166c40c26fc5cb6dcb2614a00b2becbf5c92